internal/static/cartogram2.js

```diff
@@ -2011,6 +2229,13 @@ class Cartogram {
             this.generateSVGDownloadLinks();
             this.displayVersionSwitchButtons();
             this.updateGridDocument(mappack.griddocument);
+            this.model.map.getTotalValuesForVersion(this.model.current_sysname);
```

Comment:
Do you need these function calls to `getTotalValuesForVersion`, `getTotalAreaForVersion`, and `getConvenAllArea`?